tests/grpc-client-test.py

```
@@ -121,5 +121,16 @@ def test_load_purls_file(self):
         # Ensure the method returned None (indicating a failure)
         self.assertEqual(components,expected_value)
 
+    def test_grpc_generic_metadata(self):
+        grpc_client = ScanossGrpc(debug=True, req_headers={'x-api-key': '123455', 'generic-header': 'generic-header-value'})
```

Ruff (0.8.2)

Line too long (124 > 120)

(E501)